packages/app/pages/schedule.tsx

```diff
@@ -1,12 +1,23 @@
-import { Calendar, momentLocalizer } from "react-big-calendar";
+import { Calendar, momentLocalizer, View } from "react-big-calendar";
 import moment from "moment";
 import useSWR from "swr";
 import { API } from "@template/api-client";
 import { Result } from "antd";
+import styled from "styled-components";
 
-export default function Schedule() {
-  const { data, error } = useSWR(`api/v1/courses/1`, async () =>
-    API.course.get(1)
+const ScheduleCalendar = styled(Calendar)`
+  height: 70vh;
+`;
+
+type ScheduleProps = {
+  viewType: View;
+};
+
+export default function Schedule({ viewType }: ScheduleProps) {
+  const { data, error } = useSWR(
+    `api/v1/courses/1/schedule`,
+    async () => API.course.get(1),
+    { revalidateOnFocus: false }
```

Comment:
why is this false

Reply:
Think about people that leave 50 tabs open. It might be a week later when they switch back to the tab. I don't see any benefit to disabling revalidateOnFocus.

Reply:
True though that the schedule doesn't change much. This just seems like premature optimization.

Reply:
Those are good points. I no longer feel strongly either way, so I'd say it's your call @willstenzel @NEUDitao

Reply:
I agree with @dajinchu on this one. I'll change it to not disable `revalidateOnFocus`